### PR TITLE
Add hover profile to mentions

### PR DESCRIPTION
**Pull request by mxstbr**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- hyperion (frontend)

**Release notes for users (delete if codebase-only change)**
- Hover profiles on mentions!

**EDIT**: This works now!!!!

### CODE REVIEW

File: src/components/rich-text-editor/style.js [hunk 1]
```diff
@@ -4,6 +4,65 @@ import styled, { css } from 'styled-components';
...
+      <div onMouseEnter={this.hover(true)} onMouseLeave={this.hover(false)}>
+        <Manager>
+          <Reference>
+            {({ ref }) => <Link to={`/users/${username}`}>{children}</Link>}
+          </Reference>
+          {this.state.hovered && (
+            <Popper
+              placement="top-start"
+              modifiers={{
+                preventOverflow: { enabled: false },
```
Comment (FezVrasta) [inline src/components/rich-text-editor/style.js]: you probably want to keep this on?

Comment (mxstbr) [reply src/components/rich-text-editor/style.js]: Still happens with `preventOverflow` set to `enabled: true` :confused: (I copy and pasted this from the other place we use react-popper)

File: src/components/rich-text-editor/style.js [hunk 2]
```diff
@@ -4,6 +4,65 @@ import styled, { css } from 'styled-components';
 import Link from 'src/components/link';
 import { Transition, zIndex } from '../globals';
 import { theme } from 'src/components/theme';
+import { Manager, Reference, Popper } from 'react-popper';
+import HoverProfile from 'src/components/avatar/hoverProfile';
+import { getUserByUsername } from 'shared/graphql/queries/user/getUser';
+
+const MentionHoverProfile = getUserByUsername(
+  props =>
+    console.log(props) ||
```
Comment (alejandronanez) [inline src/components/rich-text-editor/style.js]: Oops, you forgot to remove this one :)

File: src/components/rich-text-editor/style.js (hunk 1 above)
Comment (FezVrasta) [reply src/components/rich-text-editor/style.js]: I'll be able to play with it during the weekend if it's okay to you.

On top of my head I may suggest to define the `boundariesElement` to make sure it's set to the right container, but it should be done automatically by popper.

**Review by brianlovin — APPROVED** [review]
Noice!

**Pull request merged**